### PR TITLE
Project setup update.

### PR DESCRIPTION
**Pull request by AlexPolyakovsky**

### CODE REVIEW

File: src/main/java/com/appcharge/server/service/impl/OfferServiceImpl.java [hunk 1]
```diff
@@ -57,6 +71,8 @@ public ResponseEntity<?> createOffer() throws IOException {
                     JsonNode.class
             );
             JsonNode responseBody = response.getBody();
+            updateOfferId();
+            assignNewOrderToPlayers(responseBody);
```
Comment (yoav-ac) [inline src/main/java/com/appcharge/server/service/impl/OfferServiceImpl.java]: Why we are using this? I think we deleted it on the huuuge mocker.

File: Dockerfile [hunk 2]
```diff
@@ -17,17 +17,23 @@ RUN gradle clean build --no-daemon -x test
 # Create a new Docker image with the Java runtime only
 FROM adoptopenjdk:11-jre-hotspot
 
+ARG offers_path
+ARG player_dataset_path
+ARG APP_PORT
+
+ENV APP_PORT=$APP_PORT
+
```
Comment (yoav-ac) [inline Dockerfile]: @natalia-appcharge @AlexPolyakovsky  why do we need docker file here? Didnt we say that CD will be only in the mocker repo?

Comment (natalia-appcharge) [reply Dockerfile]: yes, CD will be only in the mocker repo
But how publishers will build and run code from this repo?

File: src/main/java/com/appcharge/server/service/impl/OfferServiceImpl.java (hunk 1 above)
Comment (AlexPolyakovsky) [reply src/main/java/com/appcharge/server/service/impl/OfferServiceImpl.java]: fixed

**Pull request merged**